.github/workflows/pr-reviewer-noti.yml

```
@@ -0,0 +1,67 @@
+name: PR Reviewer Notification
+
+on:
+  pull_request:
+    types: [review_requested]
+
+jobs:
+  specific_review_requested:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)